nulib/dc-nextjs
# Add placeholder texts to AI search

## What does this do?
- Changes Search `input` element to `textarea` element
- `textarea` placeholder text changes based on whether AI chat is engaged
- Put in some language on AI results when no prompt has been entered

![image](https://github.com/nulib/dc-nextjs/assets/3020266/7d926485-0580-4bc4-8bfe-399ed28bf383)

@davidschober @vlsrobinson @mathewjordan  Let me know about the placeholder text?

## How to test
- Verify placeholder texts
- Try copying or writing a lot of text into the search input to validate its a `textarea` and will wrap vertically.
